k8s/client.go

```diff
@@ -826,6 +855,8 @@ func ExtractGroupVersionResource(obj runtime.Object) (schema.GroupVersionResourc
 		Version:  gvk.Version,
 		Resource: gvk.Kind + "s",
 	}
+	fmt.Println("GVR", gvr)
```

Comment:
I don't think we should be printing this here.

Reply:
e2e is done here https://github.com/chronosphereio/calyptia-cloud-e2e/pull/171